.github/workflows/gh-pages.yml

```
@@ -32,17 +32,17 @@ jobs:
+      - name: Build
+        run: |
+          yarn build:ghpage
+
       - name: Deploy
         uses: peaceiris/actions-gh-pages@v4
         if: github.ref == 'refs/heads/main'
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./website/build
\ No newline at end of file
+          publish_dir: ./website/out
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)